### ISSUE TITLE
4.1 drop -- opening issue for tracking.

### ISSUE THREAD

**Issue by milkteacactuar** [ravahn/FFXIV_ACT_Plugin#115]
I'm sure you're probably aware and working on this already, but I just wanted to open this so people who stop by see it. Parsing is not working with the new patch etc etc.

Regards.

**Comment by ravahn** [comment]
Yes - almost ready with an update for DX11 FFXIV, doing final testing now.

**Comment by milkteacactuar** [comment]
We appreciate you! :)

**Comment by Rhess86** [comment]
Don't forget about the poor souls still using dx9 please =D
